library/core/src/mem/mod.rs

```diff
@@ -1329,6 +1380,15 @@ impl<T> SizedTypeProperties for T {}
 ///
 /// assert_eq!(mem::offset_of!(NestedA, b.0), 0);
 ///
+/// #[repr(u8)]
+/// enum Enum {
+///     A(u8, u16),
+///     B { one: u8, two: u16 },
+/// }
+///
+/// assert_eq!(mem::offset_of!(Enum, A.0), 1);
```

Comment:
Seems that enums are not actually currently supported. I'm confused, then, about the existing example below. How is this passing CI?

```rust
/// # #[cfg(not(bootstrap))]
/// assert_eq!(mem::offset_of!(Option<&u8>, Some.0), 0);
```

Reply:
Seems that adding `# #[cfg(not(bootstrap))]` fixed it.